task.sql

```diff
@@ -2,0 +1,40 @@
+CREATE DATABASE ShopDB;
+USE ShopDB;
+CREATE TABLE Products (
+     ID INT AUTO_INCREMENT,
+     Name VARCHAR(50),
+     Description VARCHAR(100),
+     Price INT,
+     WarehouseAmount INT,
+     PRIMARY KEY (ID)
+);     
+     
+CREATE TABLE Customers (
+     ID INT AUTO_INCREMENT,
+     FirstName VARCHAR(50),
+     LastName VARCHAR(50),
+     Email VARCHAR(50),
```

Comment:
It's recommended to add a `UNIQUE` constraint to the `Email` column to ensure that each customer has a unique email address.